.github/workflows/aws-integration-gpu.yaml

```
@@ -40,21 +40,43 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v4
     
-    # Issue: https://github.com/canonical/data-science-stack/issues/116
-    - name: Setup operator environment
-      run: sudo -E bash .github/workflows/setup_environment.sh
-      env:
-        MICROK8S_CHANNEL: 1.28/stable
-
-    - name: Setup microk8s
-      run: sudo -E bash .github/workflows/configure_microk8s.sh
-      env:
-        MICROK8S_ADDONS: "storage dns rbac gpu"
+    - name: Install Python and Tox
+      run: |
+        export DEBIAN_FRONTEND=noninteractive
+        sudo apt-get update -yqq
+        sudo apt-get install -yqq python3-pip
+        pip3 install tox
 
-    - name: Run tests as root
+    - name: Run tests
       run: |
-        sudo snap alias microk8s.kubectl kubectl
-        tox -e integration-gpu -- -vv -s
+        sudo snap install kubectl --classic
+        sudo snap install k8s --classic --channel=1.32-classic/stable
+        sudo k8s bootstrap
+        sudo k8s enable local-storage
+
+        mkdir -p ~/.kube
+        sudo k8s config | sudo tee ~/.kube/config > /dev/null
```

Couldn't the following work as we do in #203? 
```suggestion
        sudo k8s config > ~/.kube/config
```